### DISCUSSION TITLE
flashing sdcard issue

### DISCUSSION THREAD

**Discussion by aozdemir** [OE4T/tegra-demo-distro#166] [Q&A] [answered]
Hi I am trying to this demo image for jetson nano 4GB. My device boot up but error messages. I will write all my step can you please say my wrong or missing step.

```
$ git clone https://github.com/OE4T/tegra-demo-distro.git
$ git checkout dunfell
$ cd tegra-demo-distro
$ git submodule update --init
$ . ./setup-env --machine jetson-nano-devkit
$ bitbake demo-image-full
```

bitbake finish without any error.I extract demo-image-full-jetson-nano-devkit.tegraflash.tar.gz
`$ sudo ./dosdcard.sh /dev/sdc `
and it finish without any error. sd is 16GB .I try to boot jetson nano with this sd card.
I am getting that error message:

```
Waiting for /dev/mmcblk0p...
Mounting /dev/mmcblk0p...
mount: mounting /dev/mmc/blk0p on /mnt failed: no such file or directory
sh: can't access tty; job control turned off
```

**Comment by madisongh** [comment, marked as answer]
It looks like you haven't updated the bootloader partitions in the SPI flash.  As mentioned [on the wiki page](https://github.com/OE4T/meta-tegra/wiki/Flashing-the-Jetson-Dev-Kit#spi-flash-on-sdcard-based-kits), you'll need to do that at least once to get U-Boot and the other boot partitions set up.

**Reply by aozdemir** [reply]
thanks
